packages/openapi-generator/src/parser/media-type.ts

```diff
@@ -51,13 +54,13 @@ export function parseMediaType(
 ): OpenApiSchema | undefined {
   const allMediaTypes = getMediaTypes(bodyOrResponseObject);
   if (allMediaTypes.length) {
-    const jsonMediaType = parseApplicationJsonMediaType(
+    const parsedMediaType = parseTopLevelMediaType(
```

Comment:
[q] Why is it "top level"?

Reply:
Since we are adding more than just application/json now, I thought using the [standard term](https://www.rfc-editor.org/rfc/rfc2046.html#section-2) for it was more appropriate.

Reply:
Interesting! never heard of it.